CMakeLists.txt

```diff
@@ -123,8 +124,11 @@ set(SWIFT_INSTALL_COMPONENTS "${_SWIFT_KNOWN_INSTALL_COMPONENTS}" CACHE STRING
 # * toolchain-dev-tools -- install development tools useful in a shared toolchain
 # * dev -- headers and libraries required to use Swift compiler as a library.
 
-set(SWIFT_SDKS "" CACHE STRING
-    "If non-empty, limits building target binaries only to specified SDKs (despite other SDKs being available)")
+set(SWIFT_CROSS_COMPILE_STDLIB_TARGETS "" CACHE STRING
+    "A list of deployment-targets to cross-compile the standard-library for (by default, the standard library is always compiled for the host)")
+
+set(SWIFT_SKIP_HOST_STDLIB "" CACHE STRING
+  "If True, will not build the standard library for the host")
```

Comment:
Oh, I misunderstood. But yes, I think I encountered failures similar to the ones you mention here when I was working on #3027 -- and those were just the assumptions that the host \*stdlib\* was being built.

In any case, sounds good to me! Sorry for the noise.